src/daft-scan/src/lib.rs

```diff
@@ -500,6 +502,21 @@ impl From<ScanTask> for ScanTaskLikeRef {
 
 pub type ScanTaskRef = Arc<ScanTask>;
 
+lazy_static! {
```

Comment:
Favor OnceCell or OnceLock instead: https://doc.rust-lang.org/core/cell/struct.OnceCell.html

Reply:
Sounds good, addressing in https://github.com/Eventual-Inc/Daft/pull/3939